src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md

```diff
@@ -2,14 +2,21 @@
 
 - [ ] Structure a site using semantic HTML to aid accessibility
 - [ ] Ensure a web page is readable for screen readers
-- [ ] Design a UI without relying solely on colour, so that we don't exclude colour-blind users
 - [ ] Ensure our UI has sufficient colour contrast so that everyone can perceive it comfortably
 - [ ] Use various tools to check that our website meets accessibility criteria
 
 ## Design
 
-- [ ] **Ensure our website displays well on screens of different sizes**
-- [ ] Use CSS media queries to ensure our content is always presented effectively
-- [ ] Demonstrate a mobile-first approach to designing a website with a great user experience
-- [ ] **Create an attractive and accessible colour palette for our project**
+- [ ] Use CSS media queries to ensure our content is always presented effectively on screens of different sizes
+- [ ] Demonstrate a mobile-first approach to building a website
 - [ ] Use CSS variables to apply repeated colours to HTML elements
+- [ ] Use CSS Flex to style children in a single-direction layout (ie a row or a column)
+- [ ] Use CSS Grid to style children in two-direction layout
```

Comment:
3 Dimensional? i.e. Rows and Columns

Are we asking them to use both Flexbox and Grid; or either of them?